homeassistant/components/home_connect/number.py

```diff
@@ -136,12 +137,17 @@ def _get_entities_for_appliance(
 def _get_option_entities_for_appliance(
     entry: HomeConnectConfigEntry,
     appliance: HomeConnectApplianceData,
+    entity_registry: er.EntityRegistry,
 ) -> list[HomeConnectOptionEntity]:
     """Get a list of currently available option entities."""
     return [
         HomeConnectOptionNumberEntity(entry.runtime_data, appliance, description)
         for description in NUMBER_OPTIONS
         if description.key in appliance.options
+        # Restore it if once existed
```

Comment:
I suggest you remove this comment. The code should be self-explanatory (if not, then simplify the code :) ).

Reply:
Since the check for option entity exists is performed in several places (number, select, etc.) why not having an auxiliary function for this in `common.py`?

Reply:
I'm going to leave the comment, as I think that a comment doesn't do any damage (and con provide additional information and context to self-explanatory code), although now it is part of the new function docstring.

About your function suggestion, I like it, implemented already in the new commits.👍